src/routes/email/guards/email-registration.guard.ts

```diff
@@ -41,8 +41,8 @@ export class EmailRegistrationGuard implements CanActivate {
     const safe = request.params['safeAddress'];
     const emailAddress = request.body['emailAddress'];
     const signer = request.body['signer'];
-    const signature = request.body['signature'];
-    const timestamp = request.body['timestamp'];
+    const signature = request.headers['safe-wallet-signature'];
+    const timestamp = request.headers['safe-wallet-signature-timestamp'];
```

Comment:
Nitpick: the documentation for this class need to be adjusted (lines 26-27)

Reply:
[674050f](https://github.com/safe-global/safe-client-gateway/pull/1117/commits/674050f5060e1c16a288a23f3b7da8aa0694280d)

This guard was added while I was working on this PR 😅 . Fixed!